src/test/resources/com/puppycrawl/tools/checkstyle/main/InputMainPropertyChaining.properties

```diff
@@ -0,0 +1,2 @@
+mycheckstyle.severity=${severity}
+severity=warning
```

Comment:
what about case:
```
mycheckstyle.severity=${severity}
severity=${mycheckstyle.severity}
```

Reply:
Wow, good catch. Should this throw something like "Unresolvable property: " exception?

Reply:
Added.